test/Performance/benchmark/sqlsrv/regular/SqlsrvUpdateBench.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use SqlsrvPerfTest\SqlsrvUtil;
+/**
+ * @BeforeMethods({"connect", "setTableName", "createTable", "generateInsertValues", "insertWithPrepare", "generateUpdateValues", "generateUpdateParams"})
+ * @AfterMethods({ "dropTable", "disconnect"})
+ */
+class SqlsrvUpdateBench{
+
+    private $conn;
+    private $tableName;
+    private $insertValues;
+    private $updateValues;
+    private $updateParams;
+
+    public function setTableName(){
+        $this->tableName = "datatypes_".rand();
+    }
+
+    public function connect(){
+        $this->conn = SqlsrvUtil::connect();
+    }
+
+    public function createTable(){
+        SqlsrvUtil::createCRUDTable( $this->conn, $this->tableName );
+    }
+
+    public function generateInsertValues(){
+        $this->insertValues = SqlsrvUtil::generateInsertValues();
+    }
+
+    public function insertWithPrepare(){
+        SqlsrvUtil::insertWithPrepare( $this->conn, $this->tableName, $this->insertValues );
+    }
+    
+    public function generateUpdateValues(){
+        $this->updateValues = SqlsrvUtil::generateUpdateValues();
+    }
+
+    public function generateUpdateParams(){
+        $this->updateParams = SqlsrvUtil::generateUpdateParams();
+    }
+    /**
+     * Each iteration inserts a row into the table, updateWithPrepare() updates that row 1000 times.
+     * Note that, every update calls prepare, bindParam and execute APIs.
+     */
+    public function benchUpdateWithPrepare(){
+        for( $i=0; $i<1000; $i++ ){
```

Comment:
Seems like this 1000 is used at a lot of places. Do you think it's a better idea if you store this number as a variable in SqlsrvUtil and PDOSqlsrvUtil?